DEPENDENCIES

```diff
@@ -0,0 +1,1541 @@
+npm/npmjs/-/which/1.3.1, , approved, CQ17757
```

Comment:
We should not add this file.

Reply:
(categorically deny)

Reply:
could you elaborate more on this?

Reply:
The DEPENDENCIES file is a raw file and includes libraries which is mark as restricted.

For example: 
``` npm/npmjs/-/remark-parse/8.0.2, MIT, restricted, clearlydefined ```
https://github.com/eclipse/che-dashboard/pull/188/files/666a835e875dddbcf4c016496f0d5f48cd22b5da#diff-61bf096785eee9146e2f0c81f24e6fa49dd91e0712d8a86e7e654588c104fa1cR190

Wich is added in the file  ```che-dashboard/.deps/EXCLUDED/dev.md``` as 

```
| `remark-parse@8.0.2` | [CQ22418](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22418) |
```

We can use existing ```che-dashboard/.deps/prod.md``` and ```che-dashboard/.deps/dev.md``` files instead of adding one more DEPENDENCIES file  wich will be include the same data.